tests/gplates_1_3/screen-output

```diff
@@ -38,7 +31,7 @@ Number of degrees of freedom: 10,656 (6,528+864+3,264)
    Solving Stokes system... 30+16 iterations.
 
    Postprocessing:
-     RMS, max velocity:                  0.0256 m/year, 0.102 m/year
+     RMS, max velocity:                  0.0257 m/year, 0.0769 m/year
```

Comment:
I have run the tests now for a once-refined and also a twice-refined mesh, but unfortunately, the differences before and after the patches remain almost the same. The numbers for the max. velocity in both versions get both slightly higher, so the difference stays more or less the same.